README.md

```diff
@@ -1,10 +1,16 @@
-# Inputs/outputs
+# HTTP Codec v0.0.8
 
-## protocol
+This microservice can encode and decode HTTP messages
+
+## Configuration
+
+This codec has no settings
```

Comment:
I think we need to point that dictionary is not needed for this mecroservice

Reply:
Technically yes, but core part which its based on requires dictionary. This requirement will be gone when it'll be migrated on `th2-codec` 4.+